src/main/sensors/gyroanalyse.c

```diff
@@ -263,40 +263,46 @@ static FAST_CODE_NOINLINE void gyroDataAnalyseUpdate(gyroAnalyseState_t *state,
             float fftWeightedSum = 0;
             float dataAvg = 0;
             float dataMax = 0;
-            float dynFiltThreshold = 0;
-            bool fftIncreasing = false;
+            float dataThreshold;
```

Comment:
No use for defining it here when it's only used at line 283

Reply:
OK thanks, moved the define to where it's used.
Rebased to master.
Any help you can give as to why Travis keeps failing?